source/mainnet/net/release-notes/release-notes-lp.rst

```diff
@@ -461,15 +461,19 @@ Desktop Wallet
 
 .. _rn-ledger-app:
 
-Concordium Ledger app
+Concordium LEDGER app
 ---------------------
 
-    March 31, 2023
+    August 10, 2023
 
-    Concordium Ledger App v3.1.0 now supports Ledger Nano S Plus firmware version 1.1.0.
+        Concordium is pleased to announce support for the Concordium LEDGER app in LEDGER Live. You can install and update the Concordium LEDGER app from LEDGER Live. For details, see :ref:`Install the Concordium LEDGER app<install-ledger>`. In addition, Concordium's LEDGER app now supports the LEDGER Nano X as well as the LEDGER Nano S and LEDGER Nano S Plus.
```

Comment:
I suppose we should also mention what is new in version `4.1.2` here. It can be found in the CHANGELOG (https://github.com/Concordium/concordium-ledger-app/blob/main/CHANGELOG.md), but a summation would be something like:

- Improved a number of UI flows to require fewer clicks.
- Amounts are now prefixed with `CCD` to indicate the unit of the amount.

We also removed governance transactions and the old baker transactions, but I don't think that is relevant to the regular users so perhaps not worth mentioning here.